quarkusissue.java

```diff
@@ -175,12 +174,16 @@ void print(Map<String, String> results) throws Exception {
 
         out.println(table);
 
-        System.out.println("Copied markdown version to clipboard.");
-        StringSelection stringSelection = new StringSelection(tableBuilder.build().toString());
-        Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
-        clipboard.setContents(stringSelection, null);
+        final String markdownTable = tableBuilder.build().toString();
+        if (printMarkdown != null && printMarkdown) {
```

Comment:
```suggestion
        if (printMarkdown) {
```

Using primitive will be false by default and can't be null.